src/wallet/db.cpp

```diff
@@ -56,12 +56,12 @@ void CDBEnv::Reset()
     fMockDb = false;
```

Comment:
same here. this time thou there were no `BOOST_FOREACH` instances in the first place.